Topic tests failing

```
[ERROR] com.provectus.kafka.ui.topics.TopicTests  Time elapsed: 0.483 s  <<< ERROR!
java.lang.NoSuchMethodError: 'java.lang.String io.netty.util.internal.ObjectUtil.checkNonEmptyAfterTrim(java.lang.String, java.lang.String)'
	at com.provectus.kafka.ui.topics.TopicTests.beforeAll(TopicTests.java:27)
```

--- a/kafka-ui-e2e-checks/pom.xml
+++ b/kafka-ui-e2e-checks/pom.xml
@@ -11,7 +11,7 @@
 
     <artifactId>kafka-ui-e2e-checks</artifactId>
     <properties>
-	<kafka-ui-contract>${project.version}</kafka-ui-contract>
+        <kafka-ui-contract>${project.version}</kafka-ui-contract>
         <junit.version>5.7.0</junit.version>
         <aspectj.version>1.9.6</aspectj.version>
         <allure.version>2.13.7</allure.version>
@@ -31,6 +31,8 @@
         <allure.screendiff-plugin.version>2.13.9</allure.screendiff-plugin.version>
         <maven.surefire-plugin.version>2.22.2</maven.surefire-plugin.version>
         <allure-maven.version>2.10.0</allure-maven.version>
+        <kafka.version>3.0.0</kafka.version>
+        <netty.version>4.1.69.Final</netty.version>
     </properties>
 
     <dependencies>
@@ -43,7 +45,87 @@
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka_2.13</artifactId>
             <version>${kafka.version}</version>
+            <exclusions> <!-- could be removed when kafka version will contain zookeeper with netty 4.1.69 -->
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-buffer</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-common</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-codec</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-handler</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-resolver</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-transport</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-transport-native-epoll</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-transport-native-unix-common</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
+
+        <!--
+        whole netty dependency block could be removed
+        when kafka version will contain zookeeper with netty 4.1.69
+        -->
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-buffer</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-common</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-codec</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-resolver</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-epoll</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-unix-common</artifactId>
+            <version>${netty.version}</version>
+        </dependency>
+
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>

--- a/pom.xml
+++ b/pom.xml
@@ -18,9 +18,9 @@
         <jackson-databind-nullable.version>0.2.1</jackson-databind-nullable.version>
         <org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
         <org.projectlombok.version>1.18.20</org.projectlombok.version>
-		<org.projectlombok.e2e-checks.version>1.18.20</org.projectlombok.e2e-checks.version>
+        <org.projectlombok.e2e-checks.version>1.18.20</org.projectlombok.e2e-checks.version>
         <git.revision>latest</git.revision>
-        <zookeper.version>3.5.7</zookeper.version>
+        <zookeper.version>3.7.0</zookeper.version>
         <kafka-clients.version>2.8.0</kafka-clients.version>
         <node.version>v14.17.1</node.version>
         <dockerfile-maven-plugin.version>1.4.10</dockerfile-maven-plugin.version>
@@ -32,7 +32,6 @@
         <openapi-generator-maven-plugin.version>4.3.0</openapi-generator-maven-plugin.version>
         <swagger-annotations.version>1.6.0</swagger-annotations.version>
         <springdoc-openapi-webflux-ui.version>1.2.32</springdoc-openapi-webflux-ui.version>
-        <kafka.version>2.8.0</kafka.version>
         <avro.version>1.9.2</avro.version>
         <confluent.version>5.5.1</confluent.version>
         <apache.commons.version>2.2</apache.commons.version>

